simplifies READMEs and links to AVS docs (#19)
aerospike/proximus-examples: quote-semantic-search/README.md

performs a query a vector embedding for the provided text is generated and
Proximus performs Approximate Nearest Neighbor(ANN) search to find relevant results.

## Pre-requisites
You don't have to know Aerospike to get started, but you do need the following:

1. A Python 3.10 - 3.11 environment and familiarity with the Python programming language (see [Setup Python Virtual Environment](./prism-image-search/README.md#setup-python-virtual-environment)).

> 1. An Aerospike Vector Search host (Sandbox, or local)

Reviewer: ```suggestion
1. An Aerospike Vector Search host (sandbox or local).
```

> ## Configure Proximus Host

Reviewer: ```suggestion
## Configure AVS (Proximus) host
```

> If you are connecting to a sandbox trial you'll need to set the following. 

Reviewer: ```suggestion
If you are connecting to a trial sandbox, you'll need to set the following:
```

```shell
export PROXIMUS_HOST=<SANDBOX_IP>
```

## Start the application

```shell
cd quote-search && \\
 waitress-serve --host 127.0.0.1 --port 8080 --threads 32 quote_search:app
```

## Performing a quote search
Navigate to http://127.0.0.1:8080/search and perform a search for quotes based on a description. 

## Install using docker compose

> If you have a license key, you can easily setup Aerospike, Proximus, and the quote-semantic-search

Reviewer: ```suggestion
If you have a license key, you can easily set up Aerospike, AVS, and the quote-semantic-search
```
